### ISSUE TITLE
send question to recipient functionality

### ISSUE THREAD

**Issue by davidmooreppf** [opengovernment/askthem-legacy#21]
Issue for 8(f) in dev roadmap. When a Q reaches threshold :: 
1. If state/city elected official with a plaintext email, deliver over email. DRM can write copy. 
### 
1. If federal member of Congress, or elected official without a plaintext email address,

a) If member has verified Twitter acc't, deliver over Twitter 

b) if member does not have verified Twitter, display prompt to user to sumbit manually 
### 
1. If public figure with verified Twitter acc't, deliver over Twitter. 
### 

If no response in a week, send reminder email. Enhancements after launch will include more specific reminders to staffpeople and administrative dashboards for verified responders & elected officials.

**Comment by derekeder** [comment]
_If public figure with verified Twitter acc't, deliver over Twitter._

This warrants it's own issue in my opinion. Would require setting up a Twitter account for AskThem, and using a [gem like this](https://github.com/sferik/twitter) to hook in to the app.

_If no response in a week, send reminder email. Enhancements after launch will include more specific reminders to staffpeople and administrative dashboards for verified responders & elected officials._

This should also be its own issue. We will need to set up a separate process that keeps track of how long ago the threshold was met, and if a response was posted. Should probably come after #18 is done. This may be something we want to do manually at first.

**Comment by walter** [comment]
Feel free to split off to new issues with a reference to this issue in each of their descriptions.

I would say the Twitter sending is a next iteration (i.e. after question asking/sharing/answering cycle is complete with email as a delivery).

Agreed about sending after a week. It can probably be done manually for a period of time, but good probably worthwhile to capture as an issue while it is fresh in our minds.

**Comment by derekeder** [comment]
Core email sending to person implemented with #53. 

Spun off additional features in to issues #60 and #61.
